### DISCUSSION TITLE
point_0701

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#256] [Roy]
"20190315T160019_20190315T161019_T17RMN"
![image](https://user-images.githubusercontent.com/54947195/147530679-423dc7c8-65ab-45a8-95d6-f33d884fafd6.png)

"20190114T160619_20190114T160808_T17RMN"
![image](https://user-images.githubusercontent.com/54947195/147530729-8f3138b2-dc14-4478-8728-e9618822ddcf.png)

"20190330T155901_20190330T160533_T17RLN"
![image](https://user-images.githubusercontent.com/54947195/147530875-41e75612-0b74-45ce-9e04-50a04a959d14.png)

"20200413T155901_20200413T160427_T17RLN"
![image](https://user-images.githubusercontent.com/54947195/147531093-45f9b1c9-1479-42c1-9fcc-2c6996d050fe.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20190315T160019_20190315T161019_T17RMN"
![Captura](https://user-images.githubusercontent.com/86502862/153041684-db8dd331-9d25-46f6-8de4-4e19864b02dd.JPG)
